practice/rain.py

```diff
@@ -4,139 +4,99 @@
 # 1. Setup
```

Comment:
It looks like you already have some commits of `rain.py` in your `master` branch. In the future, try to only do work on the branch for the problem. All good here, though.